packages/blade/src/components/BottomSheet/BottomSheet.native.tsx

```diff
@@ -77,7 +79,7 @@ const _BottomSheet = ({
     getTopOfTheStack,
   } = useBottomSheetStack();
   const currentStackIndex = getCurrentStackIndexById(id);
-  const zIndex = 100 - currentStackIndex;
+  const bottomSheetZIndex = zIndex - currentStackIndex;
```

Comment:
Please test things out manually once, since bottomsheet has limited tests right now.

Reply:
yup. Also I don't really change the value. It's vscode's variable rename thing + zIndex has same value now too